### ISSUE TITLE
metagenomic sample + biosyntheticSPAdes errors

### ISSUE THREAD

**Issue by drelo** [ablab/spades#586]
I am trying to assemble metagenome while also running biosyntheticSPAdes.
I am dividing the workload between 2 clusters. I already obtained good results but for some samples in one of the clusters I am getting some warnings and metagenomes with a small size (i.e. 54 Mb in this case).

Example of log with error

```
===== Copy files finished.


===== Assembling finished.


===== Breaking scaffolds started.


== Running: /usr/local/bin/python /usr/local/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py --result_scaffolds_filename lab/andres/WASTE/OUT/MIG1-18/scaffolds.fasta --misc_dir lab/andres/WASTE/OUT/MIG1-18/misc --threshold_for_breaking_scaffolds 3


===== Breaking scaffolds finished.


===== Terminate started.


===== Terminate finished.

 * Corrected reads are in lab/andres/WASTE/OUT/MIG1-18/corrected/
 * Assembled contigs are in lab/andres/WASTE/OUT/MIG1-18/contigs.fasta
 * Domain graph is in lab/andres/WASTE/OUT/MIG1-18/domain_graph.dot
 * Gene cluster sequences are in lab/andres/WASTE/OUT/MIG1-18/gene_clusters.fasta
 * BGC cluster statistics lab/andres/WASTE/OUT/MIG1-18/bgc_statistics.txt
 * Assembled scaffolds are in lab/andres/WASTE/OUT/MIG1-18/scaffolds.fasta
 * Paths in the assembly graph corresponding to the contigs are in lab/andres/WASTE/OUT/MIG1-18/contigs.paths
 * Paths in the assembly graph corresponding to the scaffolds are in lab/andres/WASTE/OUT/MIG1-18/scaffolds.paths
 * Assembly graph is in lab/andres/WASTE/OUT/MIG1-18/assembly_graph.fastg
 * Assembly graph in GFA format is in lab/andres/WASTE/OUT/MIG1-18/assembly_graph_with_scaffolds.gfa

======= SPAdes pipeline finished WITH WARNINGS!

=== Error correction and assembling warnings:
 * 0:04:26.259     3G / 12G   WARN    General                 (kmer_coverage_model.cpp   : 218)   Too many erroneous kmers, the estimates might be unreliable
 * 0:06:42.689     3G / 18G   WARN    General                 (pair_info_count.cpp       : 341)   Unable to estimate insert size for paired library #0
 * 0:06:42.690     3G / 18G   WARN    General                 (pair_info_count.cpp       : 347)   None of paired reads aligned properly. Please, check orientation of your read pairs.
 * 0:06:42.694     3G / 18G   WARN    General                 (repeat_resolving.cpp      :  63)   Insert size was not estimated for any of the paired libraries, repeat resolution module will not run.
======= Warnings saved to lab/andres/WASTE/OUT/MIG1-18/warnings.log
```
I just run fastq to double check the quality of this sample and everything seems fine. It doesn't seem like a memory issue so I wonder is something else is missing here. How could I fix this? Thanks for your help

Attached is the spades.log and params

[param.txt](https://github.com/ablab/spades/files/5314108/param.txt)
[spadesMIG.log](https://github.com/ablab/spades/files/5314109/spadesMIG.log)

**Comment by 1dayac** [comment]
Hello!
I don't see any critical issues, only warnings.
Can you add regular metaspades log?
Data would be even more helpful!

Thanks,
Dmitry

**Comment by asl** [comment]
@drelo It seems the issue what we fixed in SPAdes 3.14.1. Will you please upgrade and give it a try?

**Comment by drelo** [comment]
Thanks for the help! Yeah at one cluster I didn't have those warnings and I am using 3.14.1 but on the other one I am using the installed one ["3.14.0"] and for the moment I can't ask for an upgrade at the moment.
A follow up question (or let me know if I should open a new issue) I obtained these 3 files with the biosynthetic module but now I can't visualize the .dot file with Bandage, I checked the supplementary material of the paper describing biosyntheticSPAdes but couldn't find additional info. Thanks for your support!
